bedrock/legal/templates/legal/eula/thunderbird-1.5-eula.html

```diff
@@ -0,0 +1,42 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "base-resp.html" %}
+{% block body_class %}sand{% endblock %}
+
+{% block extrahead %}
+  {% stylesheet 'legal-eula' %}
+{% endblock %}
+
+{% block page_title %}Mozilla Thunderbird End-User Software License Agreement — Version 1.5, Archived{% endblock %}
+
+{% block breadcrumbs %}
+  <nav class="breadcrumbs">
+    <a href="{{ url('mozorg.home') }}">Home</a> >
+    <a href="{{ url('legal.eula') }}">{{ _('Mozilla Foundation End-User Licensing Agreements') }}</a> >
+  </nav>
+{% endblock %}
+
+{% block content %}
+<div id="main-feature">
+  <h1 itemprop="name">Mozilla Thunderbird End-User Software License Agreement</h1>
+  <h2>Version 1.5, Archived</h2>
+</div>
+
+<div id="main-content">
+  <div class="contents">
+    <p>A SOURCE CODE VERSION OF CERTAIN THUNDERBIRD E-MAIL FUNCTIONALITY THAT YOU MAY USE, MODIFY AND DISTRIBUTE IS AVAILABLE TO YOU FREE-OF-CHARGE FROM WWW.MOZILLA.ORG UNDER THE MOZILLA PUBLIC LICENSE and other open source software licenses.</p>
```

Comment:
We should probably make web addresses like `WWW.MOZILLA.ORG` in the legal text hyperlinks, like we do in the other eula pages?